viz/cmd/root.go

```diff
@@ -27,6 +27,7 @@ const (
 	jsonOutput  = healthcheck.JSONOutput
 	tableOutput = healthcheck.TableOutput
 	wideOutput  = healthcheck.WideOutput
+	jsonPathOutput = "jsonpath"
```

Comment:
Feels a bit weird to me not to include this in the healthcheck package too (i.e. here https://github.com/linkerd/linkerd2/blob/main/pkg/healthcheck/healthcheck_output.go#L17). Bit of a lack of consistency, but I suppose it's not the end of the world, not sure how @alpeb feels about it.

Reply:
Well, it's not used in the healthcheck package, so IMO it's fine like that.